lms/static/sass/_ie.scss

```diff
@@ -40,8 +37,8 @@
   }
 
   // because ie doesn't like :last
-  .last {
-    margin-right: 0  !important;
+  .last.last {
```

Comment:
I've never heard of the double class method before! That's neat-o. Also a tremendous hack, but whatever. :smile:

I would strongly suggest that you add a comment with a link to this double class method technique -- even if the UX team is the only team who will ever see/care about this code at edX, there are lots of other people in the open source community who are looking at our code, and comments are great for making it clearer. That being said, you said in a different comment that we might be able to get rid of this file entirely -- if we can do that, that's even better than applying hacks like this.